component-api/src/main/java/org/talend/sdk/component/api/component/AfterVariables.java

```diff
@@ -83,6 +85,7 @@
             + "The functionality is for the Studio only.")
     @Target(ElementType.METHOD)
     @Retention(RUNTIME)
+    @Deprecated
```

Comment:
Would be nicer if the documentation part points towards what is expected to be used, no?

Reply:
yes, will update document after tech review